Assets/Scripts/StaminaBar.cs

```diff
@@ -24,20 +27,13 @@ public class StaminaBar : MonoBehaviour {
 	}
 
     // Move is called each step of the players movement
-    // Returns state of stamina bar (crash or not);
+    // Returns if player can move;
     public bool move()
     {
-        Debug.Log(stamina);
-        if(stamina > 0)
-        {
-            stamina--;
-            return true;
-        }
-        else
-        {
-            crash = true;
-            return false;
-        }
+        if (debug)
+            Debug.Log("Standard Restore: ");

```

Comment:
neither am i going to keep arguing with you or remove them, the debug elements need to remain in, you can turn on or off the display in the prefab.